Quickref.md

```diff
@@ -0,0 +1,170 @@
+# Reflex Quick(ish) Reference
+
+## Typeclasses
+
+Many Reflex functions operate in monadic context `X a`, where the monadic type X supports various additional typeclasses such as MonadWidget, MonadHold, or MonadSample in addition to Monad itself.  For Reflex-Dom users, the type X will generally be `Widget Spider (Gui Spider (WithWebView SpiderHost) (HostFrame Spider))`, from the Reflex-Dom main entry point:
+
+```haskell
+mainWidget :: Widget Spider (Gui Spider (WithWebView SpiderHost) (HostFrame Spider)) () -> IO ()
```

Comment:
This type signature is a bit scary and you don't actually have to use it.  In fact, the string "Spider" doesn't appear a single time in my 7k line reflex application.  I just use the simpler:

    fooWidget :: MonadWidget t m => m ()

So perhaps it would be better to change it to use this more generic and concise signature?